### ISSUE TITLE
[Feature Request] Colors for statusbar items

### ISSUE THREAD

**Issue by lonix1** [Fooxly/vscode-provision-bar#6]
Firstly, thanks for this excellent extension! :smile:

It would be nice if we could set a background color for the statusbar items. For example, "BUG" could be red, "TODO" could be green, etc.

Have a look at [this extension](https://marketplace.visualstudio.com/items?itemName=usernamehw.errorlens) which has this color feature, maybe it can be a source of inspiration?

<img src="https://user-images.githubusercontent.com/9638156/128408434-fee5bf09-f259-4401-970f-b14955fd2605.png" />

````json
"provision.keywords": {
  "NOTE": {
    "background": "HEX",          // <-----
    "foreground": "HEX",          // <-----
    // etc...
  },
  "BUG": {
    "background": "#ff0000",      // <-----
    // etc...
  }
}
````

**Comment by thom1606** [comment]
Hi there,

We will certainly take a look at this feature request!

**Comment by thom1606** [comment]
Hi there,
we have updated our package and this is now available from ProVision: Bar 4.0 and beyond!
For any help with setting up the background and foreground colouring you can check our manual via Help: ProVision
